corehq/toggles.py

```diff
@@ -367,6 +367,14 @@ def toggle_values_by_name(username=None, domain=None):
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
 
+VELLUM_BETA = PredictablyRandomToggle(
+    'vellum_beta',
+    'Use Vellum beta version',
+    TAG_ONE_OFF,
+    [NAMESPACE_DOMAIN],
+    randomness=0.1
```

Comment:
It's a tossup - less manual hassle, but you can't look at the feature flag page anymore and get an exact list of who's opted in or out. Think I'll leave it alone.

It turns out that the feature flag page does have a full list of domains with the flag turned on even for random toggles - it's just reeeeeally slow (unsurprisingly).